actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -136,7 +136,7 @@ def auto_discovery_link_tag(type = :rss, url_options = {}, tag_options = {})
         tag(
           "link",
           "rel"   => tag_options[:rel] || "alternate",
-          "type"  => tag_options[:type] || Mime[type].to_s,
+          "type"  => tag_options[:type] || Template::Types[type].to_s,
```

Comment:
`:rss` isn't registered: https://github.com/rails/rails/blob/5d1b7c3b441654e8008dcd303f5367883ec660a6/actionview/lib/action_view/template/types.rb#L15. Won't work for users using Action View on their own.